spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -91,8 +92,14 @@
 				<version>1.0.4</version>
 			</dependency>
 
-			<!--Starters-->
+			<!-- spring-cloud-gcp-starter-config -->
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-context</artifactId>
+				<version>${spring-cloud-context.version}</version>
```

Comment:
Depending on the release version here, PTAL @spencergibb @joaoandremartins 
Is there a better way to use a managed version of  `org.springframework.cloud:spring-cloud-context` ?

Reply:
@jabubake what do you mean?

Reply:
@joaoandremartins  : sorry for lack of context.
@spencergibb suggested using https://mvnrepository.com/artifact/org.springframework.cloud/spring-cloud-starter

Given we only need the bootstrap support, I've only used `spring-cloud-context` here.  Just confirming that this is fine and we don't need to include the starter in the `spring-cloud-gcp-dependencies` and use `spring-cloud-context` without a version in our GCP config starter.

Reply:
No version is fine, since it's in our BOM already. Also +1 for not using the starter but only what we need in this case.